RedHat-UX/red-hat-design-system
# fix(docs): remove spinner demo page from ux.redhat.com

## What I did

1. Deleted Spinner page from the Component section.


## Testing Instructions

1. Check that Spinner doesn't show up in the deploy preview link. The Component nav should look like this:
<img width="1439" alt="Screen Shot 2022-12-14 at 5 31 49 PM" src="https://user-images.githubusercontent.com/95588923/207729219-d3d02395-5a11-440e-9a80-13cbb82280e9.png">


## Notes to Reviewers
